### ISSUE TITLE
Enhance google_compute_backend_service() to enable testing of logging attributes

### ISSUE THREAD

**Issue by sukchomb** [inspec/inspec-gcp#233]
# Detailed Description

I want to be able to test that the backend service is configured for logging in a similar way as offered by the google_spanner_instance_iam_policy resource.

## Context

Being able to perform this test will further enhance the platform security posture.

**Comment by lhasadreams** [comment]
To get the logging information requires the beta api, for example:
`gcloud beta compute backend-services describe <back-end-service> --global`
@slevenick is in the final stages of switching everything over to the beta api's.
I believe that the magic module will need updating to expose the `logConfig`

**Comment by slevenick** [comment]
This will become possible once beta endpoints are added, then I will circle back on this and make sure the fields exist to expose this field

**Comment by lhasadreams** [comment]
Any update on the beta endpoints @slevenick please?

**Comment by slevenick** [comment]
Waiting on [this PR](https://github.com/inspec/inspec-gcp/pull/226) to be merged into master

It was blocked on https://github.com/inspec/inspec/issues/4925 and hopefully be merged in soon.

**Comment by slevenick** [comment]
Beta endpoints are available now via the new resources. See the `log_config` property on the backend service resource
